app/controllers/ChoiceController.scala

```diff
@@ -70,11 +71,11 @@ class ChoiceController @Inject()(
               request.declarationId match {
                 case Some(id) =>
                   updateChoice(id, choice).map { _ =>
-                    Redirect(controllers.declaration.routes.DispatchLocationController.displayPage(Mode.Normal))
+                    StandardDeclarationJorney.start
```

Comment:
:+1:

Reply:
If we are pulling this code out I think it should go into the Navigator